.github/workflows/publish-docker-containers.yml

```diff
@@ -23,25 +24,25 @@ name: Publish docker containers
         required: false
         type: choice
         options:
-          - '["explorer", "token", "trading"]'
+          # - '["explorer", "token", "trading"]'
           - '["explorer"]'
```

Comment:
Are we stopping builds for token and trading app?

Reply:
yeah, we will later switch to `nx` (https://github.com/vegaprotocol/frontend-monorepo/blob/develop/tools/ipfs-deploy.js#L106-L108) to determine which apps have been modified, this is for now just a PoC